packages/catalog-process/src/app.ts

```diff
@@ -4,8 +4,8 @@ import { logger } from "pagopa-interop-commons";
 import { z } from "zod";
 import { authMiddleware } from "./authMiddleware.js";
 import { ctx } from "./context.js";
-import eservicesRouter from "./routers/catalog.js";
-import healthRouter from "./routers/health.js";
+import eservicesRouter from "./routers/CatalogRouter.js";
+import healthRouter from "./routers/HealthRouter.js";
```

Comment:
Why are the filenames in uppercase even if the exported definitions are lowercase?

I know this is a default export, so you can name it like you want, but it's a bit strange to have uppercase filenames (which are an exception, as we normally use lowercase in TS) when the actual object inside is not uppercase.

Also, it's a bit strange to have `eservices` in the object name but `Catalog` in the file name. Maybe we can do in a later refactor though.